styles/main.css

```diff
@@ -190,11 +190,16 @@ select.search-input:disabled {
       -ms-flex-flow: column nowrap;
           flex-flow: column nowrap;
   overflow-y: hidden;
-  padding: 1.8rem 2.1rem 0 2.1rem;
+  border: 0.5px #C36C27 solid;
+  border-bottom: none;
+  border-radius: 7px;
+  padding: 0.5rem 0.5rem 0 0.5rem;
+  margin: 1.7rem 2rem 0 2rem;
 }
 @media (min-width: 600px) {
   #search-results {
-    padding: 2em 2em 0 2em;
```

Comment:
not for this PR, but should probably remove `main.css` from the repo since it's just a generated artifact from `main.scss`, right?

Reply:
agreed! should we change the gulpfile.js so that it renders the actual servable resources (js, css) out to a `dist` directory? and then link the `index.html` files to those in the dist folder? or what's the smartest way to leverage gulp.js on this front?

Reply:
Maybe after we merge some of the existing pr's we can make a new one with these changes?